forge-gui/res/adventure/common/custom_cards/sigil_of_torment.txt

```diff
@@ -0,0 +1,7 @@
+Name:Sigil of Torment
+ManaCost:no cost
+Types:Artifact
+A:AB$ Destroy | Cost$ 3 B T PayShards<5> | ValidTgts$ Creature | TgtPrompt$ Select target creature | SubAbility$ DBLifeGain | SpellDescription$ Destroy target creature. You gain 3 life. Exile Sigil of Torment.
```

Comment:
need ActivationLimit$ 1 to prevent avoiding the exile . 

Also I guess it's intended to start on the battlefield (as opposed to command zone ?).  This would make this quite strong with sacrifice/artifact synergies (like metalcraft, though it doesn't seem overpowered just very strong)


@jjayers99 What do you think about the power level of this item ?